### DISCUSSION TITLE
New Mantainers/Collaborators

### DISCUSSION THREAD

**Discussion by rtritto** [mongo-express/mongo-express#1559] [General]
Propose yourself or someone to become a maintainer or a collaborator.

---

First start to help with issue (open new one or comment with solution) and/or create PR.

Whether feedback from activities are good, you will be promoted as an official Collaborator of Mongo Express repository. By this way, you can edit/close issues/PRs and much more.

**Comment by rtritto** [comment]
FYI @dozoisch @shakaran @BlackthornYugen

**Comment by rtritto** [comment]
@xTudoS

**Reply by rtritto** [reply]
@dozoisch @shakaran @BlackthornYugen any update for @xTudoS ?

**Reply by xTudoS** [reply]
would be a honor to be part of mongo-express collaborator's team

**Reply by rtritto** [reply]
@xTudoS you should have been added, welcome! 🎉

**Reply by xTudoS** [reply]
Thank you so much!

**Comment by parjanyamodi** [comment]
Would like to collaborate and work on this project, Can someone guide me on the necessary steps to do so?

@rtritto

**Reply by rtritto** [reply]
I can't do anything, we are waiting @dozoisch @BlackthornYugen to add roles/members.
We are using this Discussion to track requests.

**Reply by xTudoS** [reply]
while so, you could get some issues / failed PR's and try to fix them. Even though you are not a directly member (like me), PR's from anyone are welcome here.

**Reply by rtritto** [reply]
@parjanyamodi are you still interested?

**Comment by rtritto** [comment]
FYI @dev-002

**Comment by JuanMaRuiz** [comment]
I'm interested

**Reply by rtritto** [reply]
How would you like to help?

As the first comment says, you can start with:
> First start to help with issue (open new one or comment with solution) and/or create PR.

**Comment by shakaran** [comment]
The issue here is that we don't have any admin permission, only write push from @BlackthornYugen and the author seems misssing. I have write push, but no admin. Plus we dont have access to packagist for syncing or docker hub

**Comment by lkoniecki** [comment]
Any update here? The latest version has significant vulnerabilities (e.g. CVE-2024-3566). There are many collaborators/dependabot-driven PRs with dependency updates that are not merged.

**Comment by rtritto** [comment]
Released v1.1.0-rc-4.
The unmerged PRs need to be reviewed.
Any help is welcome.
